oca_dependencies.txt

```diff
@@ -2,3 +2,4 @@
 # Add a repository url and branch if you need a forked version
 account-analytic
 web
+project https://github.com/modoolar/project project
```

Comment:
You have to remove this

Reply:
I know, I've added that line before ``project_key`` module was merged in order to pass travis and runbot tests. Now it can be removed :)